examples/pytorch/translation/run_translation.py

```diff
@@ -608,11 +620,6 @@ def compute_metrics(eval_preds):
         compute_metrics=compute_metrics if training_args.predict_with_generate else None,
     )
     trainer.add_callback(TBTrainerCallback)
-    # Mlflow initial
-    #set the os enviroment for MLflowCallback
-    os.environ["DISABLE_MLFLOW_INTEGRATION"] = "False"
-    os.environ["HF_MLFLOW_LOG_ARTIFACTS"]="False"
-    os.environ["MLFLOW_FLATTEN_PARAMS"]="True"
```

Comment:
Are these env vars necessary? Can we pass them on testbench in each `cmd.env` instead of here?

Reply:
Since those variables are optional, we can pass them via `cmd.env`, that's why I remove them in the scripts